codes/quantum/qubits/monitored_random_circuits.yml

```diff
@@ -0,0 +1,94 @@
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'Finite rate \cite{arXiv:2007.03822}. Can think of as a stabilizer code whose parameters depend on time, \( [[n,k(t),d(t)]] \)'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders:
+    - 'The dynamics of the monitored random circuit can be recast in the language of stabilizer codes \cite{arXiv:1903.05124}. The stabilizers of the monitored Clifford circuit can be recursively defined as \( Stab$^{(t)} = \langle P_{\tau}, Stab^{(\tau - 1)} \rangle \). The number of stabilizers increases if and only if the next Pauli \( P_{\tau+1} \) commutes with the operators in \( Stab$^{\tau} \) . If not, then the Pauli operators which do not commute with  \( P_{\tau} \) are removed and \( P_{\tau} \) is added. The likelihood of randomly generating a \( P_{\tau} \) which commutes with all the Pauli’s in the stabilizer is low when the stabilizer group is already large (at entanglement equilibrium).'
+
+  # transversal_gates:
+
+  # general_gates:
+
+  # decoders:
+
+  # fault_tolerance:
+
+
+  threshold:
+    - 'Above the critical measurement rate \( p_c\), the natural error correction properties of the circuit can no longer protect the information. This can be interpreted as the code threshold.'
+
+
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #some_other_field:
+  # - 'detail'
+
+realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  - 'Measurement induced quantum phases have been realized in trapped-ion processor \cite{arXiv:2106.05881}'
+
+notes:
+  - 'Monitored random circuits exhibit measurement induced phase transitions between systems with volume law entanglement and area-law entanglement.'
+  - 'Connectinos to information scrambling in black hole physics as introduced in Section 11 of \cite{arXiv:1903.05124}. In particular, monitored random circuits can be viewed as the Hayden-Preskill recovery problem \cite{doi:10.1103/PhysRevD.100.086001} running backwards in time. In this setting, the volume-law entanglement phase of the monitored circuit is describes the phase when information can be recovered from an old black hole (ie, a black hole that is maximally entangled with the early universe). '
+  - 'Mapping monitored random circuits to statistical mechanics models can help estiamte thresholds and code distances for these systems \cite{arXiv:2007.03822}'
+  - 'Toplogical order in (2+1)D monitored random circuits \cite{arXiv:2011.06595}'
+
+relations:
+  parents:
+    - code_id: random_codes
```

Comment:
@mgullans, parent is dynamic_gen?

Reply:
If you want to be more fine-grained, then you can make Random circuit codes a child of dynamic_gen and the parent would be Random circuit codes.  A child of dynamic_gen that is not a random circuit are the Floquet codes of Hastings and Haah.

Reply:
Thanks, I was just thinking the same thing. Parent here is random_circuit, I've made a stub.

Reply:
Added random_circuit as a parent